### PR TITLE
Jdbc integrate

### PR DESCRIPTION
**Pull request by beichenRoot**
The modification of tg-jdbc-driver module is based on https://github.com/tigergraph/ecosys/tree/master/tools/etl/tg-jdbc-driver
master branch. The latest commit id is 608125796287bc571d4c81606134bb6617d2443b. 

The following work has been done:
1.  Delete tg-jdbc-common&tg-jdbc-restpp sub module and merge the code into tg-jdbc-driver
2. Modify the code and test description in tg-jdbc-examples
3. Complete the tg-jdbc-examples test based on tigergraph-3.4.0-offline.tar.gz

### CODE REVIEW

File: tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java [hunk 1]
```diff
@@ -89,26 +110,6 @@ public static void main( String[] args ) throws SQLException {
       } catch (SQLException e) {
         System.out.println( "Failed to createStatement: " + e);
       }
```
Comment (TanYongSql) [inline tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java:112]: Can we keep one example for loading jobs? e.g., "loadMember", or "loadCompany"

File: tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/Builtins.java [hunk 2]
```diff
@@ -27,21 +27,21 @@ public static void main( String[] args ) throws SQLException {
...
       port = Integer.valueOf(args[1]);
+
       properties.put("debug", args[2]);
+
+      // Specify the graph name, especially when multi-graph is enabled.
+      properties.put("graph", args[3]);
     }
 
     /**
+     * Running on TigerGraph V3.4
```
Comment (TanYongSql) [inline tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/Builtins.java:44]: Please remove "Running on TigerGraph V3.4", it's supposed to be compatible with all versions.

File: tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java [hunk 3]
```diff
@@ -12,35 +12,45 @@
 import java.sql.Statement;
 
 /**
+ * Running on TigerGraph V3.4
```
Comment (TanYongSql) [inline tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java:15]: Please remove "Running on TigerGraph V3.4", it's supposed to be compatible with all versions.

File: tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java (hunk 1 above)
Comment (beichenRoot) [reply tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java:111]: Example of loading job is in file [.../tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/UpsertQuery.java]
e.g., "loadPost"

**Pull request closed without merging**